interactive-repl.md

```diff
@@ -39,11 +39,13 @@ on the toolbar:
  
 ![Escape](./media/RTVS-REPL-escape.png)
 
-While you are executing code in you are building up context (i.e., global
-variables, function definitions, library loads etc.). This context is
-collectively called a Workspace, and you have the ability to load and save
-workspaces. To save a Workspace using a specific filename (the default is
-.RData), click on the Save Workspace button in the REPL:
+## Workspaces and sessions
```

Comment:
I agree. But we would have to change a lot of things to fix this naming collision. As it is, the two terms "session" and "workspace" are pretty close; one is a runtime construct, the other is a static construct. If we get questions from users, we should clarify. But for now, I'm *hoping* it works. We'll see